asreview/webapp/src/PreReviewComponents/PreReviewZone.js

```diff
@@ -135,7 +146,7 @@ const PreReviewZone = (props) => {
         fetchProjectInfo();
     }
 
-  }, [state.new, props.project_id]);
+  }, [state.new, props]);
```

Comment:
Does this result in many component updates?

Reply:
It might in the future, maybe better to pass the requested variables and functions (the latter is a bit more complicated but doable).

Reply:
Turns out to be the case in multiple places. Let's resolve this in another PR.